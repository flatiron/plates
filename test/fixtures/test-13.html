<table class="names"><tr><td class="first"></td><td class="last"></td></tr></table>
