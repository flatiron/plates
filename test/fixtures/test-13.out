<table class="names"><tr><td class="first">John</td><td class="last">Smith</td></tr><tr><td class="first">Jane</td><td class="last">Doe</td></tr></table>
